types/react-native/index.d.ts

```diff
@@ -9100,9 +9100,12 @@ export namespace Animated {
 
 // tslint:disable-next-line:interface-name
 export interface I18nManagerStatic {
-    isRTL: boolean;
+    getConstants: () => { isRTL: boolean, doLeftAndRightSwapInRTL: boolean };
     allowRTL: (allowRTL: boolean) => {};
     forceRTL: (forceRTL: boolean) => {};
+    swapLeftAndRightInRTL: (swapLeftAndRight: boolean) => {};
```

Comment:
Shouldn’t this be `(flipStyles: boolean) => void` (more focused on the `void` than the name of the argument)? I guess `allowRTL` and `forceRTL` should be changed as well?

Reply:
@ca057, I'm agreed with you, but because of the `forceRTL` and `allowRTL` argument name, I decide the `swapLeftAndRightInRTL` function argument be like them.

Now, who can decide which names are better? I think Das ist nicht wichtig, Der Name im Typ ist mir wichtig. I do anything to have it in my app.

Reply:
Ah, what I meant is that those functions return `undefined` in the source code you linked and the proper return type would then be `void`. Maybe @alloy can judge if this is too nitpicky from my side? :)

Anyway, I think also adapting the name similar to what’s used in the source code and adding tests for this would be perfect

Reply:
@ca057, I don't understand what did you mean but with your second comment I've got it, so I changed it right now and I believe that is better to use `void` instead of empty curly brace. Dankeschön.

Reply:
Looks good! Can we add some tests to https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8957957bb3d7e1797486c03962d31324c0b4ab62/types/react-native/test/index.tsx ?

Reply:
@ca057, Sure, but I don't know how to start the _test_ buy your given link, I cannot understand the relation between **test**, **type** and my changes, If I can do anything please tell me.

Reply:
In the file I linked you can import the interface you worked on if it’s not already imported and create a simple test by using it with the correct types, e.g. `I18nManager.swapLeftAndRightInRTL(true)` to ensure that the usage is kept in future revisions.

Reply:
@ca057, Thanks for your help, I add the test of `I18nManager` in the given link file and push it, it doesn't exist and I add it at the end of the file.

Reply:
@ca057, Sorry, but who will merge this PR?